src/collections/_documentation/guides/migration/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Moving on Up - Switch to Hosted Sentry
+sidebar_order: 6
+---
+
+Sentry offers a hosted cloud solution in addition to an open-source one, both are functionally the same. Despite open-source being a core value for us at Sentry, it isn't necessarily recommended for everyone. As Sentry evolves, many customers are finding out that hosting Sentry on-premise can quickly become expensive to maintain, scale, and support, making our SaaS product, the better and less costly option.
+
+For more information take a look at our post on [Self Hosted or Cloud Sentry?](https://sentry.io/_/resources/about-sentry/self-hosted-vs-cloud/)
+
+## Migrating to Cloud
+
+ Organizations that start with hosting Sentry on-premise, invest time and effort in configuring and customizing their Sentry Org. Once they decide to move to a SaaS account, it's important for them to know that they can migrate a lot of that investment into their new account instead of starting from scratch.
+
+To make this migration as seamless as possible, Sentry has a migration workflow that allows you to migrate most of your account's settings and configurations including project settings, alert rules, enabled data scrubbers, team settings, saved search queries, integration platform metadata (Sentry 10), etc.
+
+One thing to note is that we won't be able to transfer over historical event data, but new incoming events will populate the projects in your new organization. You will also need to re-invite all your members and add them to the correct teams.
+
+## How to Migrate
+
+### 1. Sign up
+
+Before you begin the migration process, [sign up](https://sentry.io/signup/) for a new organization in Sentry.
+
+![Sign Up]({% asset guides/migration/01.png @path %})
+
+Once you complete the onboarding steps, grab your **org slug** from the browser URL or under `Settings > General Settings`. We'll need this info to know where to migrate your data to.
+
+![Org Slug]({% asset guides/migration/02.png @path %})
+
+> Note: You can modify your org slug in the org settings.
+
+### 2. Choose a plan
+
+Take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Notice that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or interested in our Enterprise capabilities (priority support and SLAs, single-tenant options, etc.), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
+
+### 3. Export your data
+
+Sentry open-source installs a [Command-Line](https://docs.sentry.io/server/cli/) script that allows you to perform various operations that are unachievable within the web UI. One of those is `export`, exporting your data into a transport JSON.
```

Comment:
Since you are only referring to the Docker-based command, it may make more sense to say the following:
```suggestion
Sentry open-source provides a [command line interface](https://docs.sentry.io/server/cli/) that allows you to perform various operations that are unachievable within the web UI. One of those is `export`, exporting your data into a transport JSON.
```

Reply:
I think "installs" was not accurate here as there won't be a `sentry` command with the Docker-based install as we discussed earlier.